client/sites/staging-site/components/staging-site-card/card-content/staging-site-loading-bar-card-content.tsx

```diff
@@ -1,28 +1,19 @@
+import { ProgressBar } from '@wordpress/components';
 import { useTranslate } from 'i18n-calypso';
-import { ProgressBar } from 'calypso/components/progress-bar';
 import './style.scss';
 
 type CardContentProps = {
 	isReverting: boolean;
 	isOwner: boolean;
-	progress: number;
 };
 
-export const StagingSiteLoadingBarCardContent = ( {
-	isReverting,
-	progress,
-	isOwner,
-}: CardContentProps ) => {
+export const StagingSiteLoadingBarCardContent = ( { isReverting, isOwner }: CardContentProps ) => {
 	{
 		const translate = useTranslate();
 		if ( isReverting ) {
 			return (
 				<>
-					<ProgressBar
-						key="delete-loading-bar"
-						progress={ progress }
-						className="staging-site-card__loading-bar"
-					/>
+					<ProgressBar key="delete-loading-bar" className="staging-site-card__loading-bar" />
```

Comment:
Do we even need the `key` if in both instances the progress bar is indeterminate?

Reply:
That's a good point. Thanks for pointing that out.

This key is a leftover from the old implementation. It does not seem to be necessary so I will remove it.

Reply:
Resolved in https://github.com/Automattic/wp-calypso/pull/101230/commits/ffdb0ad3b953c2a9f9b5a264153298d8d891951a.